### DISCUSSION TITLE
Launching coqui.ai 🚀

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#1647] [Announcements]
⚡️⚡️⚡️⚡️ Huge news from Coqui ⚡️⚡️⚡️⚡️
 
We launched TTS for Creators 🚀 bringing Voice Cloning to all creatives and content creators  👩‍💻🎨💻
 
⚡️ For the first time ever, you don’t need to code to use best-ever Voice Clones ⚡️

Sign up and try it out today 👇 

https://coqui.ai/

**Comment by ghost** [comment]
This is great, but why is an organization required. I don't work at a company, so I just put N/A, but is that OK with you guys on your end? I'm just confused as to why you would require a company here, considering you're targeting this at creators.

**Reply by erogol** [reply]
N/A is okay 👍
